modules/engine/src/geometry/gpu-geometry.ts

```diff
@@ -107,13 +107,6 @@ export function getAttributeBuffersFromGeometry(
   const attributes: Record<string, Buffer> = {};
   for (const [attributeName, attribute] of Object.entries(geometry.attributes)) {
     let name: string = attributeName;
-    // TODO Map some GLTF attribute names (is this still needed?)
-    switch (attributeName) {
```

Comment:
I agree we should remove this, but it will likely break a bunch of examples. We might want to consider removing this in a separate PR, get the website with all examples running again.